tethys_gizmos/templatetags/tethys_gizmos.py

```diff
@@ -295,15 +295,15 @@ def gizmo(parser, token):
 
     Example::
 
-        {% load tethys_gizmos %}
```

Comment:
Where is this `tethys` template tag defined?

Reply:
It's defined in `tethys_sdk.templatetags`.

I have no idea why tests are failing. I can't get them to fail locally.